bulletin/index.html

```diff
@@ -34,137 +34,136 @@ <h1 class="logo-brand" itemprop="brand">MACILE</h1>
+				</ul>
+			</article>
+			<article class="buarticle">
+				<h3 class="buarticle-main--title">Summer Academy 2008</h3>
+				<p>The progam was adjusted in light of the observations made in 2007.  The duration was extended to five weeks and the curriculum was redesigned to include math, engineering technology, Spanish literacy, and activities such as chess, field trips, and guest speakers.</p>
+			</article>
+			<article class="buarticle">
+				<h3 class="buarticle-main--title">About the students</h3>
+				<p>Thirty-five students were admitted and completed the program. (MACILE targets talented students in the top 20% of their class.) There was a required orientation and a meeting with the parents .</p>
+			</article>
+			<article class="buarticle">
+				<h3 class="buarticle-main--title">Schedule and evaluation</h3>
+				<p>
+				 The school day was extended from 12:00 PM to 4:00 PM and students were provided with a healthy lunch as well as breakfast. The student's mathematical and linguistic progress was gauged through a math test and an evaluation questionnaire completed at the end of the program.
+				</p>
+			</article>
+			<article class="buarticle">
+				<img src="/images/bulletin/SA2007/itabo-2007-s.jpg">
+				<p>MACILE Summer Academy 2007 and 2008 were conduceted in July at the Itabo Elementary School in Itabo, San Critóbal, Dominican Republic.</p>
+			</article>
+			<article class="buarticle">
+				<img src="/images/bulletin/espanol-2008-4.jpg">
+				<p>Spanish teacher, Antonia Asencio, and members of her class, produly displaying resources use to improve their learning.</p>
+			</article>
+			<article class="buarticle">
+				<h3 class="buarticle-main--title">Los indios de Quisqueya</h3>
+				<h4 class="buarticle-subheading">MACILE Spanish currilum</h4>
+				<p>An innovative active approach to deepen understanding of the language through the history and folblore of the Dominican Republic. A book written exclusively for MACILE by Dr. Rosario Montelongo de Swanson.</p>
+				<p>Los indios de Quisqueya, the accompaning history book and foundation of the MACILE curriculum, was donated by the Tavares Foundation. The author, Juan Tómas Tavares Kelnes, permited its use.</p>
+			</article>
+			<article class="buarticle">
+				<img src="/images/bulletin/sa2008-spanish-s.jpg">
+				<p>Learning language and enjoying the folklore. Students dancing at the bits of the grum and güira, singing the <i>salve</i>,learning Spanish.</p>
+			</article>
+			<article class="buarticle">
+				<img src="/images/bulletin/eng-tech-sa2008-s.jpg">
+				<p>Students built toothpick bridges and learned about solar energy. This team is testing their bridge resistance.</p>
+			</article>
+			<article class="buarticle">
+				<img src="/images/bulletin/group-2008-014.jpg">
+				<p>MACILE <i>nourishes the innate curiosity and wonderful imagination of young people rather than discourage them</i>, as Einstein admonished, to expand their horizons.</p>
+			</article>
```

Comment:
This doesn't read so clearly with the commas. I think using `—` would be much clearer because it wouldn't confuse what it applies to.

Reply:
Good observation. Re-wrote. I think it's clearer now.

Reply:
It's better, but still not so clear. It's a very positive sentence that is strong on its own, it doesn't benefit from the "as Einstein admonished". I think it creates a negative feel. Also the way it is written, it isn't clear what he admonished, you need to clearly separate that clause. 

MACILE strives to expand the young people's horizons, nourishing their innate curiosity and wonderful imagination; rather than discouraging them, as Einstein admonished.

or 

MACILE strives to expand the young people's horizons, nourishing their innate curiosity and wonderful imagination, rather than discouraging them, as Einstein admonished.

Reply:
But my vote is to remove it. It doesn't add anything.  Let sentence stand on its own. It is more powerful that way.

Reply:
I feel a bit uncomfortable taking implicitly taking credit for Einstein's view.  But, it might be OK to remove his name.

Consider this alternative:
Following Einstein's advice, MACILE strives to expand the young people's horizons, nourishing their innate curiosity and wonderful imagination, rather than discouraging them.